backend/docs/backend/designs/schedules/crud.md

```
@@ -0,0 +1,686 @@
+}
+```
+
+---
+
+## 3. Service 비즈니스 규칙
+
+파일: `src/modules/schedules/schedules.service.ts`
+
+### 3.1 createSchedule
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)